ring_test.go

```diff
@@ -739,3 +740,89 @@ var _ = Describe("Ring Tx timeout", func() {
 		testTimeout()
 	})
 })
+
+type fixedHash string
+
+func (h fixedHash) Get(string) string {
+	return string(h)
+}
+
+func TestRingSetAddrsAndRebalanceRace(t *testing.T) {
```

Comment:
Could you please create and move this test to `internal_test.go` that has `package redis` so we don't need to export more commands in `export_test.go`?

Reply:
Moved test and benchmark to internal_test.go